.ci/install_kata.sh

```diff
@@ -13,14 +13,21 @@ cidir=$(dirname "$0")
 source /etc/os-release || source /usr/lib/os-release
 source "${cidir}/lib.sh"
 
+FIRECRACKER="${FIRECRACKER:-false}"
```

Comment:
Nit: It might be safer to "namespace" this variable to something like `KATA_USE_FIRECRACKER` or -- better -- how about  `KATA_HYPERVISOR="firecracker"`?